site/pages/gcweb-theme/v5-migration.hbs

```diff
@@ -347,7 +347,7 @@
 			&lt;li role="presentation"&gt;&lt;a role="menuitem" tabindex="-1" href="https://www.canada.ca/en/services/culture.html"&gt;Culture, history and sport&lt;/a&gt;&lt;/li&gt;
 			&lt;li role="presentation"&gt;&lt;a role="menuitem" tabindex="-1" href="https://www.canada.ca/en/services/policing.html"&gt;Policing, justice and emergencies&lt;/a&gt;&lt;/li&gt;
 			&lt;li role="presentation"&gt;&lt;a role="menuitem" tabindex="-1" href="https://www.canada.ca/en/services/transport.html"&gt;Transport and infrastructure&lt;/a&gt;&lt;/li&gt;
-			&lt;li role="presentation"&gt;&lt;a role="menuitem" tabindex="-1" href="http://international.gc.ca/world-monde/index.aspx?lang=eng"&gt;Canada and the world&lt;/a&gt;&lt;/li&gt;
+			&lt;li role="presentation"&gt;&lt;a role="menuitem" tabindex="-1" href="https://international.gc.ca/world-monde/index.aspx?lang=eng"&gt;Canada and the world&lt;/a&gt;&lt;/li&gt;
 			&lt;li role="presentation"&gt;&lt;a role="menuitem" tabindex="-1" href="https://www.canada.ca/en/services/finance.html"&gt;Money and finances&lt;/a&gt;&lt;/li&gt;
 			&lt;li role="presentation"&gt;&lt;a role="menuitem" tabindex="-1" href="https://www.canada.ca/en/services/finance.html"&gt;Science and innovation&lt;/a&gt;&lt;/li&gt;
```

Comment:
Just remember, that line (the link) need to be udpated with the latest code. It should point to "science" and not "finance".

/cc @GormFrank

Reply:
Oh so that menu should be up-to-date too, I see.